source/_integrations/sms.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: "SMS"
+description: "SMS notification via GSM modem."
+icon: gammu.png
+ha_category:
+  - Notifications
+ha_release: 0.104.4
+ha_iot_class: Local Polling
+ha_config_flow: false
+ha_codeowners:
+  - '@ocalvo'
+---
+
+The `sms` integration allows to have a local execution SMS notification via [Gammu](https://wammu.eu/gammu/). This is ideal when the internet is offline or when the power goes out.
+
+This integration provides the following platforms:
+- Notify
+
+## Configuration
+
+To enable those notifications in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sms:
+  device: /dev/ttyUSB2
+
+notify:
+  - platform: sms
+    name: sms_person1
+    phone_number: !secret notify_sms_person1
+  - platform: sms
+    name: sms_person2
+    phone_number: !secret notify_sms_person2
+```
+And the screts yaml:
+```
+sms_person1: "+1NNNNNNNNNN"
+sms_person2: "+1NNNNNNNNNN"
+```
+Replace the NNN for the actual phone numbers
+
+To use notifications, please see the [getting started with automation page](/getting-started/automation/).
+
+It requires hassos version [3.6](https://github.com/home-assistant/hassos/releases/tag/3.6) or higher.
```

Comment:
HassOS is misspelled.
```suggestion
It requires HassOS version [3.6](https://github.com/home-assistant/hassos/releases/tag/3.6) or higher.
```

This is kinda misleading since not everybody uses HassOS, and I'm quite sure this can be done on other operating systems as well.